assets/xml/objects/object_ob.xml

```diff
@@ -1,84 +1,101 @@
+        <DList Name="gMoonChildGohtsMaskDL" Offset="0x3AD0" />
+        <DList Name="gMoonChildTwinmoldsMaskDL" Offset="0x5020" />
+        <DList Name="gMoonChildMajorasMaskDL" Offset="0x6380" />
+        <DList Name="gMoonChildMajorasMaskEyesDL" Offset="0x6730" />
+
+        <!-- Moonchild Mask Textures -->
+        <Texture Name="gMoonChildMajorasMaskTex" OutName="moonchild_majoras_mask" Format="rgba16" Width="32" Height="64" Offset="0x67E0" />
+        <Texture Name="gMoonChildMajorasMaskSpike1Tex" OutName="moonchild_majoras_mask_spike_1" Format="rgba16" Width="16" Height="16" Offset="0x77E0" />
+        <Texture Name="gMoonChildMajorasMaskSpike2Tex" OutName="moonchild_majoras_mask_spike_2" Format="rgba16" Width="16" Height="16" Offset="0x79E0" />
+        <Texture Name="gMoonChildMajorasMaskSpike3Tex" OutName="moonchild_majoras_mask_spike_3" Format="rgba16" Width="16" Height="16" Offset="0x7BE0" />
+        <Texture Name="gMoonChildMajorasMaskSpike4Tex" OutName="moonchild_majoras_mask_spike_4" Format="rgba16" Width="16" Height="16" Offset="0x7DE0" />
+        <Texture Name="gMoonChildMajorasMaskBackTex" OutName="moonchild_majoras_mask_back" Format="rgba16" Width="32" Height="32" Offset="0x7FE0" />
+        <Texture Name="gMoonChildMajorasMaskEyesTex" OutName="moonchild_majoras_mask_eyes" Format="rgba16" Width="32" Height="32" Offset="0x87E0" />
+        <Texture Name="gMoonChildOdalwasMask1TLUT" OutName="moonchild_odalwas_mask_1_tlut" Format="rgba16" Width="4" Height="4" Offset="0x8FE0" />
+        <Texture Name="gMoonChildOdalwas2TLUT" OutName="moonchild_odalwas_mask_2_tlut" Format="rgba16" Width="4" Height="4" Offset="0x9000" />
+        <Texture Name="gMoonChildGyorgsMask1TLUT" OutName="moonchild_gyorgs_mask_1_tlut" Format="rgba16" Width="4" Height="4" Offset="0x9020" />
+        <Texture Name="gMoonChildGyorgsMask2TLUT" OutName="moonchild_gyorgs_mask_2_tlut" Format="rgba16" Width="4" Height="4" Offset="0x9040" />
+        <Texture Name="gMoonChildGyorgsMask3TLUT" OutName="moonchild_gyorgs_mask_3_tlut" Format="rgba16" Width="4" Height="4" Offset="0x9060" />
+        <Texture Name="gMoonChildTwinmolds1TLUT" OutName="moonchild_twinmolds_mask_1_tlut" Format="rgba16" Width="4" Height="4" Offset="0x9080" />
+        <Texture Name="gMoonChildTwinmolds2TLUT" OutName="moonchild_twinmolds_2_tlut" Format="rgba16" Width="4" Height="4" Offset="0x90A0" />
+        <Texture Name="gMoonChildOdalwasMaskHairTex" OutName="moonchild_odalwas_mask_hair" Format="rgba16" Width="16" Height="16" Offset="0x90C0" />
+        <Texture Name="gMoonChildOdalwasMaskSkinTex" OutName="moonchild_odalwas_mask_skin_hair" Format="ci4" Width="64" Height="64" Offset="0x92C0" />
+        <Texture Name="gMoonChildOdalwasMaskEarsTex" OutName="moonchild_odalwas_mask_ears" Format="ci4" Width="64" Height="64" Offset="0x9AC0" />
+        <Texture Name="gMoonChildGyorgsMaskEyesTex" OutName="moonchild_gyorgs_mask_eyes" Format="rgba16" Width="32" Height="32" Offset="0xA2C0" />
```

Comment:
```suggestion
        <Texture Name="gMoonChildGyorgsTwinmoldsMaskEyeTex" OutName="moonchild_gyorgs_twinmolds_mask_eye" Format="rgba16" Width="32" Height="32" Offset="0xA2C0" />
```
As mentioned by Darkeye, this is used by Twinmold as well. Also it is just a singular eye.